examples/jokes/app/utils/session.server.ts

```diff
@@ -60,8 +60,7 @@ export async function requireUserId(
   request: Request,
   redirectTo: string = new URL(request.url).pathname
 ) {
-  const session = await getUserSession(request);
-  const userId = session.get("userId");
+  const userId = await getUser(request);
```

Comment:
I think you meant to use `getUserId`?

```suggestion
  const userId = await getUserId(request);
```